backend/groth16/bls12-377/mpcsetup/setup_test.go

```diff
@@ -189,12 +189,16 @@ func assignCircuit() frontend.Circuit {
 
 }
 
-func getTestCircuit(t *testing.T) *cs.R1CS {
-	return sync.OnceValue(func() *cs.R1CS {
-		ccs, err := frontend.Compile(curve.ID.ScalarField(), r1cs.NewBuilder, &Circuit{})
-		require.NoError(t, err)
-		return ccs.(*cs.R1CS)
-	})()
+var onceCircuit = sync.OnceValue(func() *cs.R1CS {
+	ccs, err := frontend.Compile(curve.ID.ScalarField(), r1cs.NewBuilder, &Circuit{})
+	if err != nil {
+		panic(err)
+	}
+	return ccs.(*cs.R1CS)
+})
+
+func getTestCircuit() *cs.R1CS {
```

Comment:
And now, we don't even need to have `getTestCircuit` function separately, we can just remove the current `getTestCircuit` function, rename `onceCircuit` variable to `getTestCircuit` and it works exactly the same.